src/main/java/uk/ac/ebi/eva/utils/TaskletUtils.java

```diff
@@ -11,13 +11,11 @@
 public class TaskletUtils {
 
     public static TaskletStep generateStep(StepBuilderFactory stepBuilderFactory, String stepName, Tasklet tasklet,
-                                           JobOptions jobOptions) {
+                                           boolean allowStartIfComplete) {
```

Comment:
The first time we did this abstraction had a lot more of job options that you had to get out... now is just one, which is only a getter... makes more sense to any that want to use the function.